tests/test_interface.rb

```diff
@@ -959,6 +967,30 @@ def test_interface_ipv4_arp_timeout
     assert_raises(RuntimeError) { nonvlanint.ipv4_arp_timeout = 300 }
   end
 
+  def test_interface_ipv4_forwarding
+    intf = interfaces[0]
+    interface_ethernet_default(intf)
+
+    i = Interface.new(intf)
+    begin
+      i.switchport_mode = :disabled
+      i.ipv4_forwarding = true
+    rescue RuntimeError, CliError => e
+      # RuntimeError when switchport_mode fails (some lc's, e.g. N7K-F248XP-25E)
+      # CliError when ipv4_forwarding fails
+      interface_supports_property?(intf, e.message)
+    end
+    assert(i.ipv4_forwarding)
+
+    i.ipv4_forwarding = false
+    refute(i.ipv4_forwarding)
+
+    i.ipv4_forwarding = true
+    assert(i.ipv4_forwarding)
+    i.ipv4_forwarding = i.default_ipv4_forwarding
+    refute(i.ipv4_forwarding)
```

Comment:
better to assert_equal here against the default instead of assuming that default will always be false.